best practice 2/PdfBill.page

```diff
@@ -0,0 +1,72 @@
+<apex:page standardController="Invoice_Statement__c"
+    standardStylesheets="false" renderAs="pdf" sidebar="false"
+    showHeader="false" applyHtmlTag="false" applyBodyTag="false">
+    
+    <html>
+		<head>
+			<apex:styleSheet value="{!URLFOR($Resource.PdfCSS,'cssPdf.css')}" />   
+        </head>
+        <body>
+            <div class="Heading">
+                <h1> <apex:outputText value="INVOICE" /> </h1>
+            </div>
+            <div>
+				<table class="invoice" id="in">
+                    <tr>
+                        <th>INVOICE #</th>
+                        <th>DATE</th>
+                    </tr>
+                    <tr>
+                        <td>{!Invoice_Statement__c.Name}</td>
+                        <td class="datelist">
+                            <apex:outputText value="{0,date,MM'/'dd'/'yyyy}">
+                                <apex:param value="{!Invoice_Statement__c.CreatedDate}" />
+                            </apex:outputText>
+                        </td>
+                    </tr>
+                </table >
+            <span class="abc">
+            <h2>ABC Containers</h2>
+                    Craig Lobo,<br />
+                    Karve Road,<br/>
+                    Pune 411004.<br/>
+                    9049980902<br/>
+                    <br /><br />
+               </span>
+                
+                
+                <table class="billto">
+                    <th>Bill To</th>
+                </table>
+                {!Invoice_Statement__c.Customer_Name__c},<br></br>
```

Comment:
, should be added only if the {!Invoice_Statement__c.Customer_Name__c} is not null